### ISSUE TITLE
Unable to access "Dashboard" navigation menu option using keyboard

### ISSUE THREAD

**Issue by krschau** [microsoft/devhome#1803]
### Steps to reproduce the bug

* Open "Dev Home" app.
* Navigate to "Machine Configuration" or "Extensions" navigation option and activate it.
* Now using keyboard try to set the focus on "Dashboard" navigation menu option.
* Observe the behavior.
https://task.ms/46742514

### Expected result

Should be able to navigate to "Dashboard" navigation menu option.

### Actual result

Keyboard focus shift to close navigation button and skips dashboard menu option.
